sessiontxn/interface.go

```diff
@@ -158,6 +158,8 @@ type TxnManager interface {
 	OnStmtErrorForNextAction(point StmtErrorHandlePoint, err error) (StmtErrorAction, error)
 	// OnStmtRetry is the hook that should be called when a statement retry
 	OnStmtRetry(ctx context.Context) error
+	// SetStmtNode saves the current stmtNode in the TxnManager
+	SetStmtNode(node ast.StmtNode)
```

Comment:
Why add a new method `SetStmtNode` instead of adding node parameter to `OnStmtStart` ?